docs/install/install.md

```diff
@@ -40,9 +40,9 @@ Note: **Install nvidia-docker before installing Blueoil.**
 
 #### Download Blueoil source code from github
 
-Clone the Blueoil repository with recursive option.
+Clone the [Blueoil repository](https://github.com/blue-oil/blueoil.git) with recursive option.
 
-    $ git clone --recursive [this repository]
+    $ git clone --recursive [Blueoil repository]
```

Comment:
@lm-lily 
Thank you for contribution.
How to write specific repo url like 

```
git clone --recursive git@github.com:blue-oil/blueoil.git
```

Reply:
@iizukak 
I intended to do as your suggestion but some people might want to use http link instead of ssh link. Is it better to directly insert the Blueoil link in the command line?

Reply:
@lm-lily I understand.